testsupport/tiers/checks.go

```diff
@@ -152,6 +156,53 @@ func (a *baseTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 		idlers(43200, "dev", "stage"))
 }
 
+type baseoneNSTierChecks struct {
+	baseTierChecks
+}
+
+func (a *baseoneNSTierChecks) GetNamespaceObjectChecks(nsType string) []namespaceObjectsCheck {
+	checks := append(commonChecks,
+		limitRange(defaultCPULimit, "750Mi", "10m", "64Mi"),
+		rbacEditRoleBinding(),
+		rbacEditRole(),
+		crtadminPodsRoleBinding(),
+		crtadminViewRoleBinding(),
+		execPodsRole(),
+		numberOfToolchainRoles(2),
+		numberOfToolchainRoleBindings(4))
+
+	checks = append(checks, commonNetworkPolicyChecks()...)
+
+	switch nsType {
+	case "dev":
+		checks = append(checks, networkPolicyAllowFromCRW(), numberOfNetworkPolicies(6))
+	}
+	return checks
```

Comment:
I wonder if we should fail here if the type is something else but `dev`?

Reply:
this is not needed - this method is called based on the values returned from `GetExpectedTemplateRefs` which is defined a few lines below. So it should not really happen that the method would be called with another type than `dev`.

Anyway, since there is only one ns type expected then this "switch case" is not really needed at all - you can drop it :-)

Reply:
Good point. Dropped in fdc2ae4e8880df89d8ee50f9ab96ee4e9f73a930